src/components/FloatingHelper/HelperContent/index.ts

```diff
@@ -1 +1,2 @@
 export * from './HelperContent';
+export * from './constants'
```

Comment:
lint - eof

Reply:
When I run `npm run lint` it shows NO error